src/org/usfirst/frc/team2399/robot/commands/ShooterStop.java

```diff
@@ -0,0 +1,45 @@
+package org.usfirst.frc.team2399.robot.commands;
+
+import org.usfirst.frc.team2399.robot.Robot;
+import org.usfirst.frc.team2399.robot.subsystems.Shooter;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class ShooterStop extends Command {
+	
+	private Shooter shooter = Robot.shooter;
```

Comment:
There aren't issues on Chloe's computer as of now. We're not sure what happened here.